<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBookDialog</class>
 <widget class="QDialog" name="AddBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>294</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Данные об авторе</string>
     </property>
     <widget class="QLabel" name="labelAddNewAuthorBack">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;a href=&quot;google.com&quot;&gt;Назад&lt;/a&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelAddNewAuthor">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;a href=&quot;google.com&quot;&gt;Добавить еще одного автора&lt;/a&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Имя:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSurname">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Фамилия:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelAddNewAuthorForward">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;a href=&quot;google.com&quot;&gt;Вперед&lt;/a&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editBookAuthorSurname">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="editBookAuthorName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Данные о книге</string>
     </property>
     <widget class="QLabel" name="labelBookName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Название:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelUDK">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>УДК Код:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCategory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Категория:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editBookName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="editUDKCode">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="editBookCategory">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Дополнительно</string>
     </property>
     <widget class="QSpinBox" name="spinBoxAmountBook">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelAmount">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введите количество экземпляров:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="indent">
       <number>-2</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelPhotoUpload">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Загрузить фотографию обложки:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="indent">
       <number>-2</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnChoosePhoto">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбрать...</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFilePath">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>backButton</zorder>
  <zorder>nextButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>groupBox_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
